### PR TITLE
TPRUN-3051 Update ant to version 1.10.12.

### PR DESCRIPTION
**Pull request by andreasmattes**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -146,7 +146,7 @@
         <framework.features.tesb.version>4.2.7.tesb3</framework.features.tesb.version>
         <static.features.tesb.version>4.2.7.tesb1</static.features.tesb.version>
         <spring.features.tesb.version>4.2.7.tesb2</spring.features.tesb.version>
-        <spring-legacy.features.tesb.version>4.2.7.tesb2</spring-legacy.features.tesb.version>
+        <spring-legacy.features.tesb.version>4.2.7.tesb3</spring-legacy.features.tesb.version>
```
Comment (jgreffe) [inline pom.xml:149]: Feature file is enterprise-legacy, not spring-legacy

**Review by jgreffe — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (andreasmattes) [reply pom.xml]: updated.

File: assemblies/features/enterprise-legacy/pom.xml [hunk 2]
```diff
@@ -31,9 +31,11 @@
     <groupId>org.apache.karaf.features</groupId>
     <artifactId>enterprise-legacy</artifactId>
     <packaging>pom</packaging>
+    <version>${revision}</version>
     <name>Apache Karaf :: Assemblies :: Features :: Enterprise Legacy</name>
 
     <properties>
+        <revision>${enterprise-legacy.feature.tesb.version}</revision>
```
Comment (jgreffe) [inline assemblies/features/enterprise-legacy/pom.xml:38]: Declared property is `enterprise-legacy.features.tesb.version` (with an `s`)

**Review by jgreffe — CHANGES_REQUESTED** [review]

File: assemblies/features/enterprise-legacy/pom.xml (hunk 2 above)
Comment (andreasmattes) [reply assemblies/features/enterprise-legacy/pom.xml]: typo corrected.

**Review by jgreffe — APPROVED** [review]

**Pull request merged**